src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -34,8 +30,8 @@ const CpsRelatedContent = ({ content }) => {
       </SectionLabel>
 
       <StoryPromoUl>
-        {content.filter(isValidItem).map(item => (
-          <StoryPromoLi key={item.id}>
+        {content.map(item => (
+          <StoryPromoLi key={item.id || item.uri}>
```

Comment:
Sorry I'm not 100% sure what this change means - so if an item doesn't have an `indexImage`, it might not have an `id`, but will have a `uri`?

Reply:
Seems related content on STY pages do not have an id.

We're not doing STYs yet, but Drew has some unit tests for an old STY PoC that was throwing a react unique keys warning - this change fixes that

Updated it to just use the URI as the key, as both MAPs and STYs have URIs

Reply:
Aha ok! Thanks for giving me the context :)

Reply:
> Updated it to just use the URI as the key, as both MAPs and STYs have URIs

Turns out the MAP URIs and STY URIs are in different paths of this object, so I reverted this again  :( I think the conditional is the best solution